GitSearch/Application/RepoInfo/RepoInfoViewController.swift

```diff
@@ -35,11 +37,15 @@ class RepoInfoViewController: UITableViewController {
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let key = props[indexPath.row]
-        let cell = tableView.dequeueReusableCell(withIdentifier: cellId, for: indexPath)
-        cell.textLabel?.text = key.replacingOccurrences(of: "_", with: " ") + ": " + repInfo[key]!
-        cell.textLabel?.numberOfLines = 0
-        return cell
+        if key == cellDescription {
+            let cell = tableView.dequeueReusableCell(withIdentifier: cellDescription, for: indexPath) as! RepoInfoDescriptionCell
+            cell.propertyValue.text = repInfo[key]!
```

Comment:
There is no need to do force unwrap here. `UILabel` will work if you set text to `nil`, but if this value is really `nil`, force unwrapping it will crash the app

Reply:
done